packages/bridge/contracts/l2/L2DaiGateway.sol

```diff
@@ -164,14 +143,20 @@ contract L2DaiGateway is L2CrossDomainEnabled {
     address to,
     uint256 amount,
     bytes calldata data
-  ) external onlyL1Counterpart(l1Counterpart) returns (bytes memory) {
+  ) external override onlyL1Counterpart(l1Counterpart) {
     require(l1Token == l1Dai, "L2DaiGateway/token-not-dai");
 
     Mintable(l2Dai).mint(to, amount);
```

Comment:
Are there any conditions in which this `mint` operation can fail?
If so, I'd suggest a try catch, and trigger an auto withdrawal to the user if this reverts

Reply:
No there is no way mint can fail if everything is setup correctly. Basically we assume that it will always work.